lib/android-helpers.js

```diff
@@ -301,6 +303,25 @@ helpers.installApk = async function (adb, opts = {}) {
   }
 };
 
+/**
+ * Installs an array of apks
+ * @param {ADB} adb Instance of Appium ADB object
+ * @param {*} opts Opts defined in driver.js
+ */
+helpers.installOtherApks = async function (otherApps, adb, opts) {
+  let {androidInstallTimeout = PACKAGE_INSTALL_TIMEOUT,
+    autoGrantPermissions} = opts;
+
+  // Install all of the APK's asynchronously
+  await B.all(otherApps.map((otherApp) => {
+    logger.debug(`Installing app: ${otherApp}`);
+    return adb.install(otherApp, {
+      grantPermissions: autoGrantPermissions,
+      timeout: androidInstallTimeout
```

Comment:
should we also consider upgrade option (`install -r`)?

Reply:
Good call. We should always replace the apps.